exercises/grep/grep_test.py

```diff
@@ -36,17 +37,55 @@
 Restore us, and regain the blissful Seat,
 Sing Heav'nly Muse, that on the secret top
 Of Oreb, or of Sinai, didst inspire
-That Shepherd, who first taught the chosen Seed
-'''
+That Shepherd, who first taught the chosen Seed'''
+FILENAMES = [
+    ILIADFILENAME,
+    MIDSUMMERNIGHTFILENAME,
+    PARADISELOSTFILENAME,
+]
+builtins.FILES = {}
 
 
-def remove_file(file_name):
-    try:
-        os.remove(file_name)
-    except OSError:
+class File(object):
+    def __init__(self, name=''):
+        self.name = name
+        self.contents = ''
+
+    def read(self):
+        return self.contents
+
+    def readlines(self):
+        return [l + '\n' for l in self.read().split('\n') if l]
```

Comment:
Can you please swap `l` for a more meaningful (and less easily confused) variable name? When I first looked at this line, I thought they were `1`s.

Reply:
Fair enough.